doc/assessment.md

```diff
@@ -117,12 +244,18 @@ Course staff see any reported issues show up on the "Issues" tab.
 
 ![Issue report](assessment-report4.png)
 
-## Access control
+## `constantQuestionValue`: Points boost for correct Homework answers
 
-See the [Access control page](accessControl.md) for details.
+FIXME
 
-By default, an assessment is only accessible to `Instructor` users. To change this, the `allowAccess` option can be used in the assessment's `infoAssessment.json` file.
+## Changing assessments while students are working on them
 
-## Adding text and links to assessments
+While it is simplest to not change an assessment while students are working on it, we know that there are sometimes good reason that changes are needed. For example, a question might need to be removed because it contains an error, new questions might be added, or the points might be changed.
 
-See the [`clientFiles` and `serverFiles`](clientServerFiles.md) page for details, and [`exam1` in the example course](https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/courseInstances/Sp15/assessments/exam1/) for an example.
+PrairieLearn’s general philosophy is to try and do the right thing in these situations. It will never automatically reduce a student’s total assessment “points” or “percentage score” due to assessment changes. Points that have been given to a student will not be taken away (unless the instructor explicitly edits the student points).
```

Comment:
Smart quotes are used for `“percentage score”`. Probably want to switch to: `"percentage score"`

Reply:
good catch, my editor was getting too clever